aiplatform/snippets/embeddings_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestGenerateEmbeddings(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	apiEndpoint := "us-central1-aiplatform.googleapis.com:443"
```

Comment:
Can the ':443" part be omitted?  it's the default https port

Reply:
In Java and Go-lang, you will be given this error message, if you omitted the port number.

rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp: address us-central1-aiplatform.googleapis.com: missing port in address"